examples/Demo/FluentUI.Demo.Client/Documentation/GetStarted/Migration/MigrationFluentSwitch.md

```diff
@@ -0,0 +1,10 @@
+---
+title: Migration FluentSwitch
+route: /Migration/Switch
+hidden: true
+---
+
+### Removed values💥
+The `ChildContent` property has been removed. Use `FluentField` instead.
```

Comment:
Can you replace with
> The `ChildContent` property has been removed. Use `Label` parameter instead.

Reply:
Ok, documentation for migration updated.